### PR TITLE
Azure Core August Releases

### PR DESCRIPTION
**Pull request by antkmsft**

### CODE REVIEW

File: sdk/core/azure-core-amqp/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
```
Comment (LarryOsterman) [inline sdk/core/azure-core-amqp/CHANGELOG.md:3]: Why no package_version.hpp change for AMQP?

Comment (antkmsft) [reply sdk/core/azure-core-amqp/CHANGELOG.md:3]: It is at the right version already.

**Review by ahsonkhan — APPROVED** [review]

File: sdk/core/azure-core-amqp/CHANGELOG.md [hunk 2]
```diff
@@ -14,10 +14,6 @@
 
 - Renamed `Azure::Core::Amqp::Models::AmqpMessageFormatValue` to `AmqpDefaultMessageFormatValue`.
```
Comment (ahsonkhan) [inline sdk/core/azure-core-amqp/CHANGELOG.md:15]: @LarryOsterman Is this AMQP changelog complete? I thought there were more changes.

Comment (antkmsft) [reply sdk/core/azure-core-amqp/CHANGELOG.md:17]: @ahsonkhan, Larry's PR just went into main, and I just updated this PR merging main to this PR, and his changes are here (expand the section above - the "Rationalized..." and "additional AMQP error values" Features are here, as well as the "changed return values" Breaking Change).

**Pull request merged**